Automattic/wp-calypso
# Atomic Revival: Display My Home notices for all three stages of revival

See https://github.com/Automattic/dotcom-forge/issues/1096

## Proposed Changes

Displays a My Home notice for each stage of Atomic Revival. They will go live once D90984-code is committed.

### Stage 1: Site automatically reverted, has no plan

![CleanShot 2022-11-01 at 07 27 51@2x](https://user-images.githubusercontent.com/36432/199258652-ed171451-d04b-4728-b8f1-4e7f7876cbc3.png)

### Stage 2: Site automatically reverted, Business plan or higher added (lower plan wouldn't display)

<img width="1124" alt="image" src="https://user-images.githubusercontent.com/36432/199261572-486cdaa1-ce46-4a08-845b-10c62f2a0f9b.png">

### Stage 3: Revived to Atomic

![CleanShot 2022-11-01 at 07 30 31@2x](https://user-images.githubusercontent.com/36432/199258758-cce40998-e525-4f0e-8f33-0c098ef72faf.png)

## Testing Instructions

1. Apply D90984-code and sandbox `public-api.wordpress.com`.
2. Create a new Free site.
3. Mock Stage 1 by adding `woa-revert-from-auto-cleanup` blog sticker: `wp blog-stickers add --sticker=woa-revert-from-auto-cleanup --who=<who> --blog_id=<id>`.
4. Visit My Home and verify the Stage 1 notice appears.
5. Purchase a Business plan for the site.
6. Visit My Home and verify the Stage 2 notice appears.
7. Activate Hosting features.
8. Visit My Home and verify the Stage 3 notice appears.
9. Restore a backup.
10. Visit My Home and verify the notice no longer appears.
